src/components/mint/collections.ts

```diff
@@ -0,0 +1,37 @@
+import { endpointKey } from 'src/config/chainEndpoints';
+
+export interface NftCollection {
+  name: string;
+  contractAddress: string;
+  networkIdx: number;
+  mintPrice: bigint;
+}
+
+// List of collections supported by Astar NFT.
```

Comment:
This list should be pulled dynamically from the graph since only supported collection will be there.

```
query CommentsByRootPostTitle {
  posts(where: {rootPost: {title_eq: "Strduster", 
                space: {id_eq: "11453"}}}) {
    id
    image
    title
    canonical
    body
  }
}
```
or more broadly with a network specific search.

```
query PostsBySpaceId {
  posts(where: {space: {id_eq: "11453"}, 
                hidden_not_eq: true, 
                tagsOriginal_containsInsensitive: "Shibuya"}, orderBy: id_DESC) {
    image
    tagsOriginal
    title
    canonical
    repliesCount
    body
    id
  }
}
```

Reply:
If it is ok with you I will merge that and address this comment later when we give a more thought about it?

Reply:
yes